### PR TITLE
List add remove

### PR DESCRIPTION
**Pull request by ForHemer**
In this project, I will implement the CRUD (create, update, delete) methods.

### CODE REVIEW

File: src/modules/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          element.id = index + 1;
+        });
+      }
+      index = tasks.length;
+
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+
+  static modifyTask(elem) {
```
Comment (Deepakdanger) [inline src/modules/addRemove.js:67]: - [x] After modifying the task, when I refresh the page, the task reset to the previous `task name`.It means the modifying task didn't get saved into `local storage`. Kindly fix this issue.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ForHemer, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addRemove.js (hunk 1 above)
Comment (ForHemer) [reply src/modules/addRemove.js:67]: @Deepakdanger Everything is ok now, I fix the issue.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ForHemer, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**